shared/src/commonMain/kotlin/com/vml/tutorial/plantshop/ui/theme/Type.kt

```diff
@@ -7,7 +7,7 @@ import androidx.compose.ui.text.font.FontWeight
 private val defaultTypography = Typography()
 val Typography = Typography(
     displayLarge = defaultTypography.displayLarge,
-    displayMedium = defaultTypography.displayMedium,
+    displayMedium = defaultTypography.displayMedium.copy(fontWeight = FontWeight.Bold),
```

Comment:
I hope this won't break some part of UI where we expect regular font weight instead of bold in the app :D

Reply:
It shouldn't be a problem for now. We can change it if it causes any problems :)

Reply:
TBH I don't know :D